roles/foreman_client_repositories/tasks/_koji_staging_repo.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: 'Foreman {{ foreman_client_repositories_version }} Client Staging Repository'
+  yum_repository:
+    name: foreman-client-koji
+    description: "Foreman {{ foreman_client_repositories_version }} Client Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-client-{{ foreman_client_repositories_version }}/{{ foreman_client_repositories_dists[ansible_os_family] }}{{ ansible_distribution_major_version }}/x86_64/"
```

Comment:
technically, we can drop the `foreman_client_repositories_dists[ansible_os_family]` and use `el` everywhere? we don't do Fedora for a while now (scope creep, sorry)

Reply:
We don't today, but we might tomorrow to try to prep for things like next RHEL so I'll just keep it since it works.